### ISSUE TITLE
Missing alt text in figures

### ISSUE THREAD

**Issue by tupui** [scientific-python/blog.scientific-python.org#33]
Some/most alt texts are missing or not providing useful information.

**Comment by tupui** [comment]
@davidevaleriani this is low priority (although mandatory for new posts), but would still be a nice thing to have at some point.

**Comment by story645** [comment]
could this maybe be a good sprint task? cc @noatamir

**Comment by jarrodmillman** [comment]
> could this maybe be a good sprint task?

@isabela-pf @MarsBarLee Thoughts?

**Comment by isabela-pf** [comment]
I think it's possibly to run an event where we fix this! @MarsBarLee and I have done this on blogs and documentation before. We have a format we can use to create a collaborative PR; does that sound good?

For me to be there, it'll have to be scheduled after April!

**Comment by tupui** [comment]
This sounds like a good idea. It's accessible and easily actionable!

**Comment by jarrodmillman** [comment]
I am busy until April 25th. Is there a good day of the week to do this (sometime in May)?

@story645  @noatamir @isabela-pf @MarsBarLee @tupui Would the afternoon of Friday, May 13th be a reasonable sprint day? If not, should we setup a whenisgood or something? Maybe @tupui would be willing to coordinate that, if the 13th doesn't work for everyone.

**Comment by tupui** [comment]
In general I should be available almost anytime for a few hours. So let me know how we should do this. Is there a conference or something we can tag on maybe? What were you thinking about? We can have a chat on discord (or slack) about that when you want, just ping me 😃

**Comment by noatamir** [comment]
May 13th works for me as well. Though I'm in CEST (UTC+2) timezone. So if you meant afternoon in PDT, that won't work for me 😞.  Maybe we should do a when2meet/whenisgood since we have a range of timezones among us?

Also, do we have a target audience for this? might help define the timezone better as well...

**Comment by MarsBarLee** [comment]
Hi all,

Thank you for your interest in a possible alt-text workshop! [Here’s an whenisgood to figure out a time](http://whenisgood.net/y5dmit3). Fill in with the times that work for you. The dates range from May 12-13, to allow for some wiggle room. I’ll close it next week.

Based on the current interest, we have people in the Americas and Europe, so around 2pm GMT could be a reasonable time for all. But depends on people’s availability.

In terms of target audience, I assumed that it would be people who responded to this thread and the SPEC discord. However, we can open it to people outside as well! We can advertise it on the SPEC social media. However, if we have people who are unfamiliar with SPEC, we’ll need to account for some time to explain what SPEC is.

Once we have enough responses and a time, I’ll send a follow-up email and Google Meet invite with more information.

Our workshops are usually 60-90 minutes. 60 minutes is the minimum to learn how to write alt-text and start contributing. If anyone needs to leave by then, that’s totally fine. 90 minutes allows some more time to work and have discussion.

**Comment by MarsBarLee** [comment]
@tupui It would be cool to have these alt-text workshops as part of a conference, but we don’t have any plans for that yet. If you know a place or could be our ‘insider’, we could propose it!

**Comment by isabela-pf** [comment]
> Also, do we have a target audience for this? might help define the timezone better as well...

Good question. My expectation was that the audience was all the people currently involved in this project and/or the blog. Based on other comments, I don't think we all share this expectation, though! This event format really can be run for anyone, so I think it's up to y'all if you want to go beyond the people already involved with this project.

> I was more thinking about newcomers since these are very low level hanging fruits

This event format is definitely newcomer friendly, especially since Git knowledge is not necessary. At the same time, not all images are easy to describe without some knowledge about the field. For example, Mars has witnessed me suffering to improvise an image description for images on the Scikit-learn docs because I had genuinely no idea what the chart was communicating. As long as we have a mix of people and images, this shouldn't be a problem. I just want to point out that even though the code changes are often single-line and clear, the images aren't always 100% newcomer friendly. We will want at least some people around who understand the information-heavy images like visualizations.

A quick review of the blog now shows me we have a mix of images, but we definitely have some doozies in the blog posts that will provide quite a challenge. Exciting! 

> I am not a UI/UX/other acronyms. But I am sure there are things to do 😃

Do you mean we should come up with other sprint tasks for the blog? Personally, I think it's better to scope the event specifically. If someone is planning a longer sprint, though, I could see the imagine for this. I wouldn't be able to help find more issues in this case, sorry.

If this does end up being looped into a conference somehow, I'd like to know as soon as we can. In my experience this often involves a very different commitment and accountability than an impromptu event for the project. I personally think it's too short notice for success, but I will back you up if it comes to that.

**Comment by story645** [comment]
> but we definitely have some doozies in the blog posts that will provide quite a challenge.

If you flag which ones, I might be able to put you in touch w/ the authors. Even if they can't come, they may be able to give more context.

**Comment by tupui** [comment]
I am sold @isabela-pf 😄 Thanks for the details!

**Comment by MarsBarLee** [comment]
Hi all! Here’s an update on the workshop.

It will be held May 13, 13:00-14:00 UTC. 

- [Click here to see this event time in your local timezone](https://arewemeetingyet.com/UTC/2022-05-13/13:00/Alt-Text%20Workshop:%20SPEC%20diagrams#eyJ1cmwiOiJtZWV0Lmdvb2dsZS5jb20vdmp0LXRkYWctd2ltIn0=)
- [Add to your Google Calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MHVla29qZm5scXEzaXBtNzEwOXI4czlyMTEgbWxlZUBxdWFuc2lnaHQuY29t&tmsrc=mlee%40quansight.com)

This is based on the whenisgood responses. Thank you for filling it out!

Here’s the [meeting agenda](https://hackmd.io/bfhftUCiTRqx2S8CTGUt6g)
- 0-5: Welcome, chit-chat, wait for people to arrive
- 5-20: Learn how to write alt text and goals for the event
- 20–50: Work time and discussion
- 50–60: Review alt text
- 60+: Optional. For any further questions

I’ll also share this in the Discord and ask the social media team if they would like to invite others who are not in this repo or Discord.

**Comment by jarrodmillman** [comment]
@story645 Would you mind helping publicize this to the matplotlib folks?
https://twitter.com/scientific_py/status/1523733881651834880

Maybe retweeting from the matplotlib account and mentioning that it will mostly involve adding alt-text to matplotlib blog posts (as most of the content is currently matplotlib content). Thanks!!!

**Comment by noatamir** [comment]
Hi Jarrod, I can also share it on the Matplotlib mailing list/discourse today. I just saw that Hannah already retweeted it yesterday.
